source/develop/release-management/features/storage/editing-floating-disks.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Editing Floating Disks
+category: feature
+authors: sleviim
+feature_name: Editing Floating Disks
+feature_modules: storage,engine,model
+feature_status: WIP 4.4
+---
+
+# Editing Floating Disks
+
+## Summary
+
+Currently, editing disks is only applicable for VM disks.
+
+This feature enables the user to edit a floating disk, and also to extend its size.
+
+### Owner
+
+*   Shani Leviim (<sleviim@redhat.com>)
+
+### Detailed Description
+
+Before this feature, only VM disks were editable (from 'VMs' -> 'Disk' sub-tab).
+Therefore, for editing a disk, regardless of the required changed parameter (VM dependent or not), it first should be attached to a VM.
+
+It is now possible editing disks from the 'Disks' tab, and also from the REST API, without prior attaching those disks to VMs, so they can remain floating.
+
+### User Experience
+
+A new 'Edit' button was added to the 'Disks' tab:
+
+![](/images/wiki/edit-floating-disks-button.png)
+
+
+Once the popup shows up, the user can edit the disk's non-VM
```

Comment:
What about disks that are attached to VMs? Can we edit them also?

Reply:
Yes we can, but there're fewer characteristics that can be edited.